app/controllers/application_controller.rb

```diff
@@ -9,6 +10,12 @@ def current_user
     @current_user = User.where(id: session[:user_id]).first
   end
 
+  def set_cache_buster
```

Comment:
I've seen you use this code before, it's probably a good idea to move it into a helper/concern/mixin module  for reusability

Reply:
Oh wait, does this PR need to be rebased?

Reply:
As far as I know, I only use it once? But I can check and change it if need be.

Also, if its merged in does it need to be rebased?